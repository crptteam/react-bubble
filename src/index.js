import Bubble from './components/Bubble';

export { Bubble };

export default Bubble;