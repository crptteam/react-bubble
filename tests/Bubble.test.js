import React from "react";
import { Bubble } from "../src/";

describe("Bubble", () => {

  it("Should renders without problems", () => {

    const wrapper = shallow(
      <Bubble>Inner message.</Bubble>
    );

    expect(wrapper).toMatchSnapshot();
  });
});

