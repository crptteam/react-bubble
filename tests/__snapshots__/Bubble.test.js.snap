// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bubble Should renders without problems 1`] = `
<BubbleWrap>
  <BackgroundWrap />
  <ContentWrap>
    <BubbleContent>
      Inner message.
    </BubbleContent>
  </ContentWrap>
</BubbleWrap>
`;
